README.md

```diff
@@ -25,9 +26,30 @@ The files committed in the `shared_files/encrypted` directory hold secret inform
 - Create a file named `.vault_password` in the root directory of the repository
 - Place the Ansible Vault password in this file
 
-#### Git hook
 
-To avoid committing and pushing unencrypted secret files, use the included `scripts/pre-commit` git pre-commit hook by running the following script from the repository root directory:
+#### Installing and Using Pre-commit
+
+Anyone committing to this repo must use the pre-commit hook to lower the likelihood that secrets will be exposed.
+
+##### Step 1: Install pre-commit
+
+You can install pre-commit using the MacOS package manager Homebrew:
+
+```sh
+brew install pre-commit
+```
+
+Other installation options can be found in the [pre-commit documentation](https://pre-commit.com/#install).
+
+##### Step 2: Install the hooks
+
+Run the following command to install the gitleaks hook:
+
+```sh
+pre-commit install
+```
+
+This will download and install the pre-commit hooks specified in `.pre-commit-config.yaml`.
 
 ```
 cp ops/pre-commit .git/hooks
```

Comment:
We could also add `go-imports` from https://github.com/TekWizely/pre-commit-golang to the `.pre-commit-config.yaml` to maintain the previous hook functionality but I'd want @kyeah's opinion on that.

Reply:
Oh that would be nice to move everything to the pre-commit.config.yaml @gfreeman-navapbc @gsf. We would definitely want to maintain the previous functionality. I assume the gitleaks config would handle the ansible vault secrets check but would want to confirm as well:

https://github.com/CMSgov/bcda-app/blob/2ab83d60a3bb007661a7fd2fad23c547096a9e3c/ops/pre-commit#L16-L29

cc. @laurenkrugen-navapbc I think we had created a ticket to improve the pre-commit, this may make it easier to do so in the future since there's a lot of built-in utilities in https://github.com/TekWizely/pre-commit-golang.